src/storage/FixedConvertingStore.ts

```diff
@@ -0,0 +1,45 @@
+import type { Representation } from '../ldp/representation/Representation';
+import type { RepresentationPreference } from '../ldp/representation/RepresentationPreference';
+import type { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
+import type { Conditions } from './Conditions';
+import type { RepresentationConverter } from './conversion/RepresentationConverter';
+import { PassthroughStore } from './PassthroughStore';
+import type { ResourceStore } from './ResourceStore';
+
+/**
+ * Store that converts incoming data when required.
+ * If the content-type of an incoming representation does not match one of the stored types it will be converted.
+ */
+export class FixedConvertingStore extends PassthroughStore {
```

Comment:
`ExcludableConvertingStore`/`BlacklistedConvertingStore`?

Reply:
Wouldn't it be a whitelist since it keeps a list of supported types?

Reply:
Depends how you look at it :P
Blacklist in the sense that the given list will _not_ be converted, but used as-is.
Given the confusion white/blacklist might not be ideal here.

Reply:
I don't like the current name, but I find it very hard to find something that's good 😄

Reply:
The recommended term has recently changed into _banlist_. (https://source.android.com/setup/contribute/respectful-code)

Reply:
I can't help myself, but such policies always remind me of Newspeak 😄

Reply:
I definitely understand it might feel weird to replace some terms we've used for years, but all little bits help. We are building the Web for all 🙂

Reply:
True, should have realized that. But I still don't like the name since the input list is not something we're banning 😄